pkgs/development/compilers/gambit/default.nix

```diff
@@ -2,6 +2,7 @@
 
 callPackage ./build.nix {
   version = "4.8.9";
+  git-version = "v${version}";
```

Comment:
Eval error because this isn't a recursive set. Try using `callPackage ./build.nix rec {` to make it recursive.

Reply:
Oops. Thanks a lot! Fixed.